Cargo.toml

```diff
@@ -26,7 +26,7 @@ crate-type = ["cdylib", "rlib"]  # rlib included just for testing
 enum-map = "2.7.3"
 itoa = "1.0.10"
 katcp-codec-fsm = { path = "crates/fsm", features = ["pyo3"] }
-pyo3 = { version = "0.21.0-beta.0", features = ["extension-module"] }
+pyo3 = { version = "0.21.0", features = ["extension-module"] }
```

Comment:
I think this also needs to be updated in the katcp-codex-fsm sub-crate?

Reply:
Good catch, thanks. Should be fixed now.